### PR TITLE
Added value attribute to TensorVariable subclasses

### PR DESCRIPTION
**Pull request by fonnesbeck**
I hate having to remember `tag.test_value` all the time. Now all variables should return the test value using a `value` property.

### CODE REVIEW

**Comment by twiecki** [comment]
That's nice syntactic sugar. However, I'm concerned that this might be confusing with the way pymc2 used `.value` -- as it's currently sampled value. With this change, `.value` would never change. Perhaps renaming this to `.init_value` would be more clear?

**Comment by fonnesbeck** [comment]
Good point. I have changed that.

TravisCI seems to be failing on `conda create` for Py2.7. Not sure why.

**Comment by fonnesbeck** [comment]
Bizarre conda failure. Only happens for 2.7, and the PATH seems correct. Any ideas?

**Comment by fonnesbeck** [comment]
Actually, my implementation is probably not good enough because whenever a function is applied to the variable it will blow away the `init_value` property. I probably need to subclass `TensorVariable` instead.

**Comment by stevenmanton** [comment]
@fonnesbeck I'm getting the same Travis build error for 2.7. I'll look into it, but can you let me know if you figure out why it's failing?

**Comment by stevenmanton** [comment]
@fonnesbeck I think I [found the error](https://github.com/stevenmanton/pymc3/commit/aaae325d2b0b5a473b31220fde88c323d3bb222b). Have a look.

**Comment by fonnesbeck** [comment]
Aha, legacy Python!

**Comment by twiecki** [comment]
Rebased off of current master.

**Comment by springcoil** [comment]
Any chance someone can rebase this and push it in again.

**Comment by twiecki** [comment]
Done.

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ python:
   - "3.4"
 
 before_install:
+  - sudo apt-get install liblapack-dev gfortran python-scipy libblas-dev libsuitesparse-dev swig
```
Comment (twiecki) [inline .travis.yml:8]: Is this still required?

**Comment by twiecki** [comment]
We should probably also merge this as it seems to pass now. I don't think we need the travis changes though.

**Comment by fonnesbeck** [comment]
Replaced by #995

**Pull request closed without merging**